.travis.yml

```diff
@@ -25,7 +17,8 @@ install:
     - python setup.py install
 
 script:
-    - coverage run -m tests.run_tests
+    # - coverage run -m tests.run_tests
+    pytest
 
 after_success:
     - coveralls
```

Comment:
codecov should be better since that it should fusion the information from different version if I am not wrong.